specification/dartLangSpec.tex

```diff
@@ -12970,16 +12970,14 @@ \subsubsection{Generic Function Instantiation}
 }
 
 \LMHash{}%
```

Comment:
If *f* is an expression with a callable object type with a generic `call`method, will "type inference" magically have inserted the `.call` at this point? (I guess it will, since the context type is a function type and the expression's static type is otherwise a callable object. Is it worth mentioning?)

Reply:
I don't think we have support for that. ['Function closurization'](https://github.com/dart-lang/language/blob/a8b6b66346d975c9d475a5fa8b43b9bf71d3cce4/specification/dartLangSpec.tex#L12853) in the language specification never mentions `.call` methods, so generic function instantiation would need to be applicable to a "callable object type" rather than a generic function type, and we don't have that either.